src/components/ContainerFluid.js

```diff
@@ -1,11 +1,8 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { Grid } from 'patternfly-react'
 
-export const ContainerFluid = ({ children }) => (
-  <div className='container-fluid'>
-    {children}
-  </div>
-)
+const ContainerFluid = ({ children }) => (<Grid fluid>{children}</Grid>)
```

Comment:
Does we really still need `ContainerFluid` component? We can remove it, and replace by `Grid` from `patternfly-react`.

Reply:
Nope, it isn't needed.  I'll remove it and swap it out for `Grid`.